app/javascript/components/app.jsx

```diff
@@ -47,9 +48,16 @@ const fragmentMatcher = new IntrospectionFragmentMatcher({
   introspectionQueryResultData,
 });
 
+const cache = new InMemoryCache({ addTypename: true, fragmentMatcher });
+
+persistCache({
```

Comment:
why do we need to persist cache to local storage?

Reply:
we wanted the world state to be stored in local storage, and since we're storing the world state in Apollo's cache, we persist